modules/virt-attaching-vm-to-ovn-secondary-nw-cli.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-attaching-vm-to-ovn-secondary-nw-cli_{context}"]                                   
+= Attaching a virtual machine to an OVN-Kubernetes secondary network using the CLI
+
+You can connect a virtual machine (VM) to the OVN-Kubernetes secondary network by including the network details in the VM configuration.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+. Edit the `VirtualMachine` manifest to add the OVN-Kubernetes secondary network interface details, as in the following example:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachine
+metadata:
+  name: vm-server
+spec:
+  template:
+    spec:
+      domain:
+        devices:
+          interfaces:
+          - name: default
+            masquerade: {}
+          - name: flatl2-overlay <1>
+            bridge: {}
+      networks:
+      - name: default
+        pod: {}
+      - name: flatl2-overlay  <2>
+        multus:
+          networkName: l2-network <3>
```

Comment:
sorry for having missed this before. Maybe - for transparency - we should also ensure the namespace appears here ?... 

This is especially important given openshift's `project` notion. 

I've checked the other examples at it seems we only list the namespace on [bridge](https://docs.openshift.com/container-platform/4.14/virt/vm_networking/virt-connecting-vm-to-linux-bridge.html#virt-attaching-vm-secondary-network-cli_virt-connecting-vm-to-linux-bridge); for [sriov](https://docs.openshift.com/container-platform/4.14/virt/vm_networking/virt-connecting-vm-to-sriov.html#virt-attaching-vm-to-sriov-network_virt-connecting-vm-to-sriov), we do not.

I think it should be aligned across all 3 . Either they all feature the namespace, or none does. Same for the procedure description.

@phoracek or for some reason, for sriov this doesn't allow specifying the namespace ? 

Should I open a doc bug to align the existing resources ?

Reply:
I would prefer to keep our example without the namespace for all. With that, people can copy paste the example and apply it directly to whatever their currently selected namespace is.

To preserve the cross-namespace example, I would suggest we explain that in a callout.

I'll leave it to @sjhala-ccs to say whether we can handle it here or should create a BZ for a follow-up PR.

Reply:
As long as they're aligned, I'm happy. FWIW, you justification for removing is IMO solid.

Reply:
I would suggest creating a GitHub issue or BZ to remove the namespace reference from the [bridge](https://docs.openshift.com/container-platform/4.14/virt/vm_networking/virt-connecting-vm-to-linux-bridge.html#virt-attaching-vm-secondary-network-cli_virt-connecting-vm-to-linux-bridge) example.

Reply:
Opened this github issue @sjhala-ccs .

Thanks.